aspnetcore-webapi/MediatrExceptionHandler/Tests/MiddlewareTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using MediatR.Pipeline;
+using MediatrExceptionHandler.Common;
+using MediatrExceptionHandler;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Newtonsoft.Json;
+
+public class MiddlewareTests
+{
+    Mock<ILogger<GlobalExceptionMiddleware<BaseRequest<SomeResponse>, SomeResponse, Exception>>> loggerMock;
```

Comment:
Those should be `private `and starting with an _

e.g. 
`private BaseRequest<SomeResponse> _request;`